Sources/Foundation/Documentation.docc/Curation/FoundationEssentials/Fundamentals/NumbersAndBasicValues/Decimal.md

```diff
@@ -0,0 +1,45 @@
+# ``FoundationPreview/Decimal``
```

Comment:
There is indeed a `Decimal`  in the repo, but that is just a stub and does nothing. As I said in another comment I think we might want to consider holding this off until we decide its fate